src/test/java/org/ambraproject/rhino/service/impl/ArticleListCrudServiceImplTest.java

```diff
@@ -0,0 +1,404 @@
+package org.ambraproject.rhino.service.impl;
+
+
+import edu.emory.mathcs.backport.java.util.Collections;
+import org.ambraproject.rhino.identity.ArticleIdentifier;
+import org.ambraproject.rhino.identity.ArticleListIdentity;
+import org.ambraproject.rhino.model.Article;
+import org.ambraproject.rhino.model.ArticleList;
+import org.ambraproject.rhino.model.Journal;
+import org.ambraproject.rhino.rest.RestClientException;
+import org.ambraproject.rhino.rest.response.ServiceResponse;
+import org.ambraproject.rhino.service.ArticleCrudService;
+import org.ambraproject.rhino.service.ArticleListCrudService;
+import org.ambraproject.rhino.view.article.ArticleRevisionView;
+import org.ambraproject.rhino.view.journal.ArticleListView;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.orm.hibernate3.HibernateTemplate;
+import org.springframework.test.context.ContextConfiguration;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.when;
+import static org.testng.Assert.fail;
+
+
+@ContextConfiguration(classes = ArticleListCrudServiceImplTest.class)
+@Configuration
+public class ArticleListCrudServiceImplTest extends AbstractStubbingArticleTest {
+  protected static final String ARTICLE_LIST_ID_TYPE = "valid-type";
+  protected static final String ARTICLE_LIST_ID_JOURNAL = "valid-journal";
+  protected static final String ARTICLE_LIST_ID_KEY = "valid-key";
+  protected static final String ARTICLE_LIST_TITLE = "article list 1";
+
+  private ArticleListCrudService mockArticleListCrudService;
+
+  private ArticleCrudService mockArticleCrudService;
+
+  private HibernateTemplate mockHibernateTemplate;
+
+  @BeforeMethod
+  public void initMocks() throws IllegalAccessException, NoSuchFieldException {
+    mockArticleListCrudService = applicationContext.getBean(ArticleListCrudService.class);
+    mockHibernateTemplate = applicationContext.getBean(HibernateTemplate.class);
+  }
+
+  @Bean
+  public ArticleListCrudService articleListCrudService() {
+    mockArticleListCrudService = spy(ArticleListCrudServiceImpl.class);
+    LOG.debug("articleListCrudService() * --> {}", mockArticleListCrudService);
+    return mockArticleListCrudService;
+  }
+
+  @Bean
+  public ArticleCrudService articleCrudService() {
```

Comment:
this bean is already declared in `AbstractStubbingArticleTest`

Reply:
removed.